exonum-java-binding/common/src/main/java/com/exonum/binding/common/proofs/list/UncheckedListProof.java

```diff
@@ -30,5 +30,5 @@
   /**
    * Returns raw source proof of this UncheckedListProof.
    */
-  ListProofNode getRootProofNode();
+  ListProof getListProof();
```

Comment:
I don't quite get what is going on here. Why isn't ListProof UncheckedListProof,
  but is contained in it?

Reply:
`ListProof` is an entity that contains `ListProofNode` and `length`. We could go with a different name for that entity, that's the best one I could come up with. I thought about `ListProofRoot` naming, but it doesn't reflect the fact that we have length in there.

Reply:
ListProof name is perfect — the question is why we have a ListProof which seems to be a complete proof from the list — and also an UnchekedListProof interface that contains the ListProof.

Reply:
Created a task for that - https://jira.bf.local/browse/ECR-3429